lib/kitto/hooks/router.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Kitto.Hooks.Router do
+  use Plug.Router
+
+  if Mix.env == :dev, do: use Plug.Debugger, otp_app: :kitto
+  unless Mix.env == :test, do: plug Plug.Logger
+
+  plug :match
+  plug :dispatch
+
+  match "/:hook" do
+    if Kitto.Hooks.hooks[String.to_atom hook] do
```

Comment:
Better alias both Kitto.Hooks and Kitto.Notifier at the top of the module definition.